packages/projects-docs/pages/learn/sandboxes/_meta.json

```diff
@@ -1,8 +1,8 @@
 {
   "overview": "Overview",
+  "your-first-sandbox": "First-Time Setup",
   "editors": "Editors",
   "embedding": "Embedding",
-  "tests": "Tests",
   "secrets": "Secrets",
   "configuration": "Configuration",
   "templates": "Templates",
```

Comment:
This content seems to be entirely related to Browser Sandboxes. So we should consider moving it there or keep it here but update the content.